raiden-ts/src/path/utils.ts

```diff
@@ -93,9 +91,9 @@ export function pfsInfo(
     withLatestFrom(config$),
     mergeMap(([url, { httpTimeout }]) => {
       if (!url) throw new RaidenError(ErrorCodes.PFS_EMPTY_URL);
-      else if (!urlRegex.test(url)) throw new RaidenError(ErrorCodes.PFS_INVALID_URL, { url });
+      else if (!isValidUrl(url)) throw new RaidenError(ErrorCodes.PFS_INVALID_URL, { url });
       // default to https for domain-only urls
-      else if (!url.startsWith('https://')) url = `https://${url}`;
+      else if (!url.startsWith('https://') && !url.startsWith('http://')) url = `https://${url}`;
```

Comment:
Any change we could also add a test for this case too?

Reply:
Done by https://github.com/raiden-network/light-client/pull/1344/files#diff-4d311b8a210fcb7f2c2090f0aec4ca2dR228

Reply:
I was referring to the different with `http`